syncer/job.go

```diff
@@ -58,8 +58,8 @@ func (t opType) String() string {
 
 type job struct {
 	tp              opType
-	sourceSchema    string
-	sourceTable     string
+	sourceSchemas   []string
+	sourceTables    []string
```

Comment:
why job need to use multiple schemas and tables? can you add a comment

Reply:
and I think map is better

Reply:
all updated in https://github.com/pingcap/dm/pull/587/commits/2733185c2787894f9104e20aab935ac154092e4b